src/Umbraco.Web.BackOffice/Filters/FileUploadCleanupFilterAttribute.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Umbraco.Core;
+using Umbraco.Core.Logging;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Web.WebApi.Filters
+{
+    /// <summary>
+    ///     Checks if the parameter is IHaveUploadedFiles and then deletes any temporary saved files from file uploads
+    ///     associated with the request
+    /// </summary>
+    internal sealed class FileUploadCleanupFilterAttribute : TypeFilterAttribute
```

Comment:
This attribute was removed from some actions in this PR but this was also moved? How come? I thought we didn't need this filter anymore?

Reply:
Some upload methods handle the stream in the action, in these cases we don't need this attribute anymore. As the stream is not saved to a temp folder that needs to be cleaned up. But we still have some examples are we use the temp folder.